packages/components/src/form-file-upload/index.tsx

```diff
@@ -18,7 +18,7 @@ import type { FormFileUploadProps } from './types';
  *
  * const MyFormFileUpload = () => (
  *   <FormFileUpload
- *     accept="image/*"
+ *     accept="image/*, image/heic"
```

Comment:
I don't think we should change this example code here

Reply:
Right, this won't be necessary after the `FormFileUpload` component update.